editor/src/com/talosvfx/talos/editor/addons/scene/assets/AssetRepository.java

```diff
@@ -1356,34 +1356,45 @@ public void deleteRawAsset (FileHandle handle) {
 
 	}
 
-	public FileHandle copyRawAsset (FileHandle file, FileHandle directory) {
-		return copyRawAsset(file, directory, false);
-	}
-
-	public FileHandle copyRawAsset (FileHandle file, FileHandle directory, boolean replace) {
-		String fileName = file.name();
-		if (directory.child(fileName).exists() && !replace) {
-			String baseName = file.nameWithoutExtension();
+	public FileHandle copyRawAsset (FileHandle file, FileHandle destination) {
+		return copyRawAsset(file, destination, false);
+	}
+
+	/**
+	 * In the first synopsis form, the copyRawAsset utility copies the contents of the source_file to the target_file.
+	 * In the second synopsis form, the contents of each named source_file is copied to the target_file target_directory.
+	 * The names of the files themselves are changed in case of collision, while replace flag is false.
+	 * @param source_file
+	 * @param target_file
+	 * @param replace
+	 * @return FileHandle of newly copied file.
+	 */
+	public FileHandle copyRawAsset (FileHandle source_file, FileHandle target_file, boolean replace) {
```

Comment:
What are these underscores in var names!?

Reply:
As far as I can tell, this is doing nothing but changing names? Please dont do this in feature PRs, needs to be reverted

Reply:
mb, it's the UNIX influence

Reply:
fixed